app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,10 +1,12 @@
+import { buildSelectOptions } from "utils";
 import * as yup from "yup";
 
 export const NOTES_LIST_DATA = [
   {
     id: 1,
     title: "How to claim the warranty?",
-    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    description:
+      '"Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn\'t getting',
```

Comment:
Not sure why you've enclosed this whole string within single quotes. This should be enclosed within double quotes. Go through the string and see if the double quotes already within the string add any value. I feel it doesn't add any value.

Reply:
I followed the figma design.  I will remove the double quotes within the string.

Reply:
Thanks for pointing that out. Will update the docs.